src/components/organisms/ChatSidebar/ChatSidebar.scss

```diff
@@ -1,6 +1,6 @@
 @import "scss/constants";
 
-$header-height: 45px;
+$header-height: 50px;
```

Comment:
Any reason for this increase? I reckon it can affect some component because not all css code is using $header-height const as per my knowledge

Reply:
That is some extra space for those new labels. 
![image](https://user-images.githubusercontent.com/16973041/131631613-71bbfaa0-3608-4450-af6c-e3b5974b290b.png)

Without it buttons look squeezed.

Regarding the UI - I've adjusted not only this but a lot of other styles in the ChatSidebar component. After all that I looked through it and haven't found any style bugs.